extensions/cornerstone/src/utils/measurementServiceMappings/EllipticalRoi.js

```diff
@@ -1,9 +1,35 @@
 import SUPPORTED_TOOLS from './constants/supportedTools';
 import getSOPInstanceAttributes from './utils/getSOPInstanceAttributes';
+import getHandlesFromPoints from './utils/getHandlesFromPoints';
 
 const EllipticalRoi = {
   toAnnotation: (measurement, definition) => {
-    // TODO -> Implement when this is needed.
+
+    /* TODO: To be finished/updated. */
+    const {
+      id,
+      label,
+      description,
+      points,
+      unit,
+      SOPInstanceUID,
+      FrameOfReferenceUID,
+      referenceSeriesUID,
+    } = measurement;
+
+    return {
+      toolName: definition,
+      measurementData: {
+        sopInstanceUid: SOPInstanceUID,
```

Comment:
Is there a reason why don't use the same casing? e.g. SOPInstanceUID  in measurementData?

Reply:
No problem, updated.